models/reporter.go

```diff
@@ -110,14 +117,16 @@ func inSlice(slice []string, val string) bool {
 // Select select metrics for report.
 func (r *Reporter) Select(ctx context.Context, periodStartFromSec, periodStartToSec int64,
 	dimensions map[string][]string, labels map[string][]string,
-	group, order string, offset, limit uint32,
+	group, order, search string, offset, limit uint32,
```

Comment:
Note for the future. 
Maybe we can create a new ticket to group all the parameters into a struct